dashboard/db/schema.rb

```diff
@@ -1393,6 +1393,16 @@
     t.index ["user_id"], name: "index_queued_account_purges_on_user_id", unique: true
   end
 
+  create_table "reference_guides", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci", force: :cascade do |t|
```

Comment:
this should be utf8, not utf8mb4. please see SETUP.md for how to fix this

Reply:
i thought i had done this already, do I need to do this again after a db:reset?

Reply:
ah, yeah, unfortunately, you would need to do it again after a db:reset.